tests/EdgeToEdge/Routes/DisplayPageRouteTest.php

```diff
@@ -19,21 +19,19 @@
 class DisplayPageRouteTest extends WebRouteTestCase {
 
 	public function testWhenPageHasCustomTemplate_customTemplateIsRendered(): void {
-		$this->createEnvironment(
-			[],
-			function ( Client $client, FunFunFactory $factory ): void {
-				$factory->setContentPagePageSelector( $this->newMockPageSelector( 'test' ) );
-				$factory->setContentProvider( $this->newVfsContentProvider( [ 'test.twig' => '' ] ) );
-
-				$crawler = $client->request( 'GET', '/page/test' );
-
-				$this->assertCount( 1, $crawler->filter( '.test-block' ) );
-			}
+		$client = $this->createClient( [ 'skin' => [ 'default' => 'cat17' ] ] );
+		//supporters.twig page from fundraising-frontend-content repository
+		$client->request(
+			'GET',
+			'/page/Unterstützerliste'
 		);
+		$crawler = $client->getCrawler();
+
+		$this->assertSame( 1, $crawler->filter( '.donors' )->count() );
```

Comment:
This is testing the output of `web_content`, not the custom markup that is done in `supports.html.twig` (adding a script tag). My suggestion: check for a script tag with an attribute ending in `supporters.js`